### PR TITLE
Linting pycubedmini.py

### PR DESCRIPTION
**Pull request by thetazero**
Currently incomplete & need to update the other 2 pycubedmini lib files.

### CODE REVIEW

File: advanced/lib/pycubedmini.py [hunk 1]
```diff
@@ -36,85 +41,84 @@
...
+            'WDT':    False,
+            'Sun':    False,
+            'Coils':  False
+        }
+        self.micro = microcontroller
 
         self.data_cache = {}
         self.filenumbers = {}
-        self.vlowbatt = 6.0
+        self.vlowbatt = 3.5
```
Comment (jbwillis) [inline advanced/lib/pycubedmini.py:88]: Not sure it really matters, but this probably shouldn't be changed in this merge.

File: advanced/lib/pycubedmini.py [hunk 2]
```diff
@@ -125,64 +129,63 @@ def __init__(self):
...
-            sun_xp = adafruit_tsl2561.TSL2561(self.i2c2,address=0x49) # +X
-            sun_sensors=[sun_zn,sun_yn,sun_xn,sun_yp,sun_xp]
+            sun_yn = adafruit_tsl2561.TSL2561(self.i2c2, address=0x29)  # -Y
+            sun_zn = adafruit_tsl2561.TSL2561(self.i2c2, address=0x39)  # -Z
+            sun_xn = adafruit_tsl2561.TSL2561(self.i2c1, address=0x49)  # -X
+
+            sun_yp = adafruit_tsl2561.TSL2561(self.i2c1, address=0x29)  # +Y
+            sun_zp = adafruit_tsl2561.TSL2561(self.i2c1, address=0x39)  # +Z
+            sun_xp = adafruit_tsl2561.TSL2561(self.i2c2, address=0x49)  # +X
+            sun_sensors = [sun_zn, sun_yn, sun_xn, sun_yp, sun_xp]
```
Comment (jbwillis) [inline advanced/lib/pycubedmini.py:159]: `sun_zp` needs to be added to this list.

Comment (jbwillis) [reply advanced/lib/pycubedmini.py:159]: Though maybe we merge the linter and then correct this later.

**Review by jbwillis — CHANGES_REQUESTED** [review]
Overall, really good. I'm excited to have this set up. There are a couple of "non-linting" changes here that I want you to comment on or change (you decide how important it is to include/exclude them).

File: advanced/lib/pycubedmini.py (hunk 1 above)
Comment (thetazero) [reply advanced/lib/pycubedmini.py:88]: fair enough.

File: advanced/lib/pycubedmini.py (hunk 2 above)
Comment (thetazero) [reply advanced/lib/pycubedmini.py:159]: Yes I'd like to do that separately and I've made an issue for it already: #17

**Comment by thetazero** [comment]
I've copy pasted in the changes into the other 2 pycubedmini files & made sure not to modify the voltage threshold for them

**Review by jbwillis — APPROVED** [review]

**Pull request merged**